features/aws-ids.yaml

```diff
@@ -1,3 +1,3 @@
-xenial: ami-06e94647aeaed1e5c
 bionic: ami-094e9c95623db463c
 focal: ami-0193aa0a9df84a08b
+xenial: ami-06e94647aeaed1e5c
```

Comment:
This alphabetical order bothers me more than it should :eyes: :upside_down_face: